scalafix/src/main/scala/org/typelevel/sbt/TypelevelScalafixPlugin.scala

```diff
@@ -28,9 +28,26 @@ object TypelevelScalafixPlugin extends AutoPlugin {
 
   override def trigger = allRequirements
 
+  object autoImport {
+    val tlTypelevelScalafixModules = settingKey[Seq[String]](
+      "The typelevel-scalafix modules to add to the scalafix dependency classpath, or Seq.empty to omit them entirely. See available modules at https://github.com/typelevel/typelevel-scalafix."
+    )
```

Comment:
How about something like this instead @armanbilge - so that users can pick modules rather than making the version optional?

Reply:
Hmm. I think the fact that rules have to be activated individually anyway suggests a single aggregate module would work fine here. Otherwise maintaining a rule list and a module list starts to be a bit much I think. (Also we'd have to keep the default in sbt-typelevel up-to-date).

Reply:
Cool, let's add an aggregate module in typelevel-scalafix then

Reply:
>  Or is this just about typelevel-scalafix's rules

It is, but we're going back on this. I think we should just publish all the rules under one aggregate module, because you still have to activate them individually in your scalafix config.

Reply:
I've reverted this change as we now depend on the aggregate module instead :D